.github/workflows/build_latest_image.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
```

Comment:
I'm not sure if it is worth to use the action. As it is open source and by Docker themselves, I hope it is trustworthy. But in fact this whole tool just becomes useful if you use a custom registry at certain cloud providers etc. Else it comes down to [this little function](https://github.com/docker/login-action/blob/b30d77254f6c8aeebb0f00776ac12447e199919d/src/docker.ts#L25), which in fact is nothing more than `docker login --username $DOCKER_USERNAME --password $DOCKER_TOKEN`. So very simple. Just as some input here...

Reply:
Good point.